Modules/Loadable/Markups/Widgets/Resources/UI/qSlicerSimpleMeasurementsWidget.ui

```diff
@@ -0,0 +1,506 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>qSlicerSimpleMeasurementsWidget</class>
+ <widget class="qSlicerWidget" name="qSlicerSimpleMeasurementsWidget">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>300</width>
+    <height>864</height>
+   </rect>
+  </property>
```

Comment:
Some of these hardcoded dimensions may be contributing to the "Delete" button cut off in the screenshot. My screen was 1080p tall and maybe you're using as 1200p tall monitor or more. I believe removing these defined width/height geometries should resolve this. There could be others in this UI file like this farther down in the code.

Reply:
thanks!  yes I could reproduce the issue by resizing Slicer, I will fix it tomorrow